libs/category/state/src/selectors/category-page/category-page.selector.ts

```diff
@@ -164,6 +171,11 @@ const createCategoryPageSelectors = <V extends DaffGenericCategory<V>>(): DaffCa
     (state: DaffCategoryReducerState) => state.errors,
   );
 
+  const selectCategoryPageMutating = createSelector(
```

Comment:
I think this name should be `selectIsCategoryPageMutating`

Reply:
changed